components/psst/BUILD.gn

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2023 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import("//brave/build/config.gni")
+
+static_library("psst") {
+  sources = [
+    "psst_component_installer.cc",
+    "psst_component_installer.h",
+    "psst_rule.cc",
+    "psst_rule.h",
+    "psst_service.cc",
+    "psst_service.h",
+    "psst_tab_helper.cc",
+    "psst_tab_helper.h",
+  ]
+
+  deps = [
+    "//base",
+    "//brave/components/brave_component_updater/browser",
+    "//brave/extensions:common",
+    "//chrome/common",
```

Comment:
this is a layering violation, you cannot include anything from `//chrome/*` in components